packages/core/src/apiml/Services.ts

```diff
@@ -207,7 +207,26 @@ export class Services {
 
         const _genCommentsHelper = (key: string, elements: string[]): string => {
             if (elements == null || elements.length === 0) return "";
-            return `//"${key}": "${elements.length === 1 ? elements[0] : elements.join('"\n//"' + key + '": "')}"`;
+            let kvPair: string = "";
+            let endComma = "";
+            // add comma to end of kvp depending on if base profile
+            if(!key.includes("base")){
+                endComma = ",";
+            }
```

Comment:
`endComma` can be made `const` with the use of the ternary operator:

```suggestion
            // add comma to end of kvp depending on if base profile
            const endComma = key.includes("base") ? "" : ",";
```